Client/metrics.yaml

```diff
@@ -325,6 +325,18 @@ tabs:
     notification_emails:
       - firefox-ios@mozilla.com
     expires: "2021-07-01"
+  add-new-tab-button:
```

Comment:
**Comment B**
Need to make sure add new button counter has been added correctly?